app/src/main/java/org/mozilla/fenix/components/BackgroundServices.kt

```diff
@@ -217,13 +217,11 @@ internal class TelemetryAccountObserver(
             metricController.track(it)
         }
 
-        // Used by Leanplum as a context variable.
         settings.fxaSignedIn = true
```

Comment:
Should we remove these settings as well?

Reply:
At first I removed it and then I was wondering whether we would want to keep it for other things. But maybe you are right, no need to keep for now, I'll get rid of it.